lib/webvalve/service_url_converter.rb

```diff
@@ -2,48 +2,76 @@
 require 'addressable/template'
 
 module WebValve
+  # @api private
```

Comment:
this reminds me, i gotta do a pass over this library to call `private_constants`. will make a note for myself for that too